src/Raven.Server/Smuggler/Documents/StreamSource.cs

```diff
@@ -206,6 +206,15 @@ public DatabaseRecord GetDatabaseRecord()
                     }
                 }
 
+                if (reader.TryGet(nameof(databaseRecord.UnusedDatabaseIds), out BlittableJsonReaderArray unusedDatabaseIds) &&
+                    unusedDatabaseIds != null)
+                {
+                    foreach (LazyStringValue id in unusedDatabaseIds)
```

Comment:
I wonder if this can be `LazyCompressedStringValue`. Would it be safer to do: `foreach (string id in unusedDatabaseIds)`?
We have implicit conversion to `string` for both cases..

Reply:
I have tried that but it throws an exception so I will change it to 
```
foreach (var id in unusedDatabaseIds)
{
    databaseRecord.UnusedDatabaseIds.Add((string)id);
}
```
Ok?

Reply:
I think it would be better either to skip it and log an error or fail the import entirely.
We are doing this before we even imported a single doc right? so let's fail it completely..